samples/BenchmarkDotNet.Samples/IntroThreadingDiagnoser.cs

```diff
@@ -0,0 +1,19 @@
+﻿using BenchmarkDotNet.Attributes;
+using System.Threading;
+
+namespace BenchmarkDotNet.Samples
+{
+    [ThreadingDiagnoser] // ENABLE the diagnoser
+    public class IntroThreadingDiagnoser
```

Comment:
Currently, this sample test is useless because `TargetFrameworks` doesn't contain `netcoreapp3.0` in `BenchmarkDotNet.Samples.csproj`. If someone would like to run this test, they have to add missing `TargetFrameworks`.

Reply:
again I was aware of that, I should have explain that in the PRs description.